app/javascript/dashboard/i18n/locale/ar/report.json

```diff
@@ -59,7 +59,56 @@
     "CUSTOM_DATE_RANGE": {
       "CONFIRM": "تطبيق",
       "PLACEHOLDER": "اختر نطاق المدة"
-    }
+    },
+    "GROUP_BY_FILTER_DROPDOWN_LABEL": "تجميع بواسطة",
+    "GROUP_BY_DAY_OPTIONS": [
+      {
+        "id": 1,
+        "groupBy": "اليوم"
```

Comment:
@aswindevps @sojan-official The key is getting translated here. I believe we need a groupBy Label and groupKey attribute